src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -84,10 +84,11 @@ protected function outputRoutes(OutputInterface $output, $routes = null)
 
         $output->writeln($this->getHelper('formatter')->formatSection('router', 'Current routes'));
 
-        $maxName = strlen('name');
-        $maxMethod = strlen('method');
-        $maxScheme = strlen('scheme');
-        $maxHost = strlen('host');
+        // strlen('name') = 4, strlen('method') = 6, strlen('scheme') = 6, strlen('host') = 4
+        $maxName = 4;
+        $maxMethod = 6;
+        $maxScheme = 6;
+        $maxHost = 4;
```

Comment:
I'm not sure saving these 4 calls is a good idea. It is not in a loop, and the previous code was much more readable IMO